fe/fe-core/src/main/java/org/apache/doris/fs/obj/ObjStorage.java

```diff
@@ -34,7 +34,7 @@
 public interface ObjStorage<C> {
 
     // CHUNK_SIZE for multi part upload
-    public static final int CHUNK_SIZE = 5 * 1024 * 1024;
+    int CHUNK_SIZE = 5 * 1024 * 1024;
```

Comment:
why removing `public static final`?

Reply:
In Java, all fields in an interface are implicitly public, static, and final, so explicitly declaring them as such is redundant.